### ISSUE TITLE
Navigation - Subsection complete/error style - 508/WCAG

### ISSUE THREAD

**Issue by Jay-Neighbours-I72** [truetandem/e-QIP-prototype#1389]
Is the current subsection complete/error style 508/WCAG compliant? 

Current subsection style:
Complete = green circle
Error = red vertical line (same vertical red line used for errors in top sections) and red circle.

<img width="414" alt="screen shot 2017-06-07 at 11 30 32 am" src="https://user-images.githubusercontent.com/19935974/26887004-e37ef3c4-4b74-11e7-8c91-7a9e857bd1f4.png">

**Comment by lauraponce** [comment]
Agreed, I have that same assumption. If someone in unable to differentiate green from red, they would have the secondary indicator of the exclamation point to show there is an issue and the vertical line to indicate _where_ the issue is occurring.
